docs/installation/bootstrap.rst

```diff
@@ -9,12 +9,13 @@ Preparation
    from the Scality repositories.
 
 #. Download the MetalK8s ISO file on the machine that will host the bootstrap
-   node. Mount this ISO file at the specific following path:
+   node. Mount this ISO file at the path of your choice (we will use
+   ``/mnt/metalk8s`` as an example for the rest of this guide):
 
    .. parsed-literal::
 
-      root@bootstrap $ mkdir -p /srv/scality/metalk8s-|version|
-      root@bootstrap $ mount <path-to-iso> /srv/scality/metalk8s-|version|
+      root@bootstrap $ mkdir -p /mnt/metalk8s
+      root@bootstrap $ mount <path-to-iso> /mnt/metalk8s
```

Comment:
Not sure we want to document this way.
To me, it's good to support this but in the documentation, we should rely on only one mountpoint everywhere so that user do not have the exact same iso mounted in 2 different places

Reply:
Is it a problem though? Mountpoints are not "expensive", and we only need to mount the first ISO manually, others are managed with `iso-manager.sh`.
If you feel strongly about this, I can revert the doc change.

Reply:
Sure it's not "expansive".
No strong opinion, but to me It's better to get the user used to always going to this `/srv/scality/` folder for every action

Reply:
Updated to use the `/srv/scality/metalk8s-|version|` mountpoint, but kept the explanation saying any mountpoint will work, as we will mount the ISO under `/srv/scality` anyway.